project-maintainers.csv

```diff
@@ -256,7 +256,6 @@ Graduated,etcd,Benjamin Wang,WMware,ahrtr,https://github.com/etcd-io/etcd/blob/m
 ,,Marek Siarkowicz,Google,serathius,
 ,,Piotr Tabor,Google,ptabor,
 ,,Sahdev Zala,IBM,spzala,
-,,Sam Batschelet,Red Hat,hexfusion,

```

Comment:
It's unrelated to this PR but seems `WMware` for Benjamin should be corrected to `VMware`.